downstream/modules/platform/proc-import-mesh-ca.adoc

```diff
@@ -0,0 +1,24 @@
+[id="proc-import-mesh-ca_{context}"]
+
+= Importing a Certificate Authority (CA) certificate
+
+A Certificate Authority (CA) is used to verify and sign individual node certificates in an {AutomationMesh} environment. Users who wish to provide their own CA can do so by specifying the path to the certificate and the private RSA key file in the `inventory` file of their {PlatformName} installer.
+
+NOTE: Users are not required to have a CA prior to their {AutomationMesh} installation. Users without a CA can proceed with their {PlatformNameShort} installation where the installer will generate a CA for the user using OpenSSL.
```

Comment:
again, my mistake, but we don't use openssl as a backend. It uses the golang crypto library to generate CA, certs, and keys. I think we can just end the statement with "for the user".